### ISSUE TITLE
XML file causes crash in RobotLog2DB

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-robotlog2db#25]
Accidently I passed an XML result file created by the Python pytest module to robotlog2db.py, instead of an Robot Framework XML result file.

This causes the following crash:

```
Traceback (most recent call last):
  File "C:\RobotTest\projects\ROBFW_AIO\robotframework-robotlog2db\RobotLog2DB\robotlog2db.py", line 1268, in <module>
    RobotLog2DB()
  File "C:\RobotTest\projects\ROBFW_AIO\robotframework-robotlog2db\RobotLog2DB\robotlog2db.py", line 1111, in RobotLog2DB
    result = ExecutionResult(*sources)
  File "D:\RobotFramework\python39\lib\site-packages\robotframework-4.1.3-py3.9.egg\robot\result\resultbuilder.py", line 52, in ExecutionResult
    return _single_result(sources[0], options)
  File "D:\RobotFramework\python39\lib\site-packages\robotframework-4.1.3-py3.9.egg\robot\result\resultbuilder.py", line 77, in _single_result
    raise DataError("Reading XML source '%s' failed: %s" % (unic(ets), error))
robot.errors.DataError: Reading XML source 'D:\Test\aiotestlogfile.xml' failed: Incompatible root element 'testsuites'.
```

But it is possible to parse the root tag of an XML file, and therefore it is possible to check if the name of the root tag is like expected.

An incompatible root tag should not cause a crash.

**Comment by HolQue** [comment]
The application also crashes if I pass a Robot Framework XML result file containing syntax errors (e.g. missing closing tags).

But syntax errors can be detected. Therefore a syntax error in XML file should not cause a crash.

**Comment by test-fullautomation** [comment]
Hi Ngoan,
please look into this.
Thank you,
 Thomas

**Comment by test-fullautomation** [comment]
solved and already merged #36
